src/reporters/JSONReporter.ts

```diff
@@ -15,7 +16,7 @@ export class JSONReporter implements IReporter {
 
   async report(practicesAndComponents: PracticeWithContextForReporter[]): Promise<void> {
     const reportString = this.buildReport(practicesAndComponents);
-    console.log(inspect(reportString, { showHidden: false, depth: null }));
+    debug.log(inspect(reportString, { showHidden: false, depth: null }));
```

Comment:
`debugLog `?

Reply:
debugLog() doesn't log in terminal. debug.log() logs both in the terminal and in the file if the -l is used.